bundles/io.openliberty.tools.eclipse.lsp4e/pom.xml

```diff
@@ -55,36 +53,48 @@
                             <goals>
                                 <goal>copy-dependencies</goal>
                             </goals>
-                            <phase>generate-resources</phase>
                             <configuration>
                                 <classifier>uber</classifier>
-                                <outputDirectory>${basedir}/server/</outputDirectory>  
+                                <outputDirectory>${project.basedir}/server/</outputDirectory>  
                             </configuration>        
                         </execution>
-                        <execution>
-				            <id>copy-libertyls</id>
-				            <phase>generate-resources</phase>
-				            <goals>
-				              <goal>copy</goal>
-				            </goals>
-				            <configuration>
-				               <artifactItems>
-				                <artifactItem>
-				                  <groupId>io.openliberty.tools</groupId>
-				                  <artifactId>liberty-langserver-lemminx</artifactId>
-				                  <version>1.0-SNAPSHOT</version>
-				                  <classifier>jar-with-dependencies</classifier>
-				                  <type>jar</type>
-				                  <overWrite>true</overWrite>
-				                  <destFileName>liberty-langserver-lemminx-1.0-SNAPSHOT-jar-with-dependencies.jar</destFileName>
-				                </artifactItem>
-				               </artifactItems> 
-				              <outputDirectory>${basedir}/lib</outputDirectory>
-				            </configuration>
-				        </execution>    
                     </executions>    
               </plugin> 
-            
+              <plugin>
+                  <groupId>org.apache.maven.plugins</groupId>
+                  <artifactId>maven-shade-plugin</artifactId>
+                  <version>3.4.0</version>
+                  <executions>
+                     <execution>
+                         <phase>process-sources</phase>
+                         <goals>
+                             <goal>shade</goal>
+                         </goals>
+                         <configuration>
+                           <artifactSet>
+                             <includes>
+                               <include>io.openliberty.tools:liberty-langserver</include>
```

Comment:
should this be: <include>io.openliberty.tools:liberty-langserver-lemminx</include>